pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel_test.go

```diff
@@ -21,6 +21,10 @@ import (
 	"net/http"
 	"testing"
 
+	"k8s.io/apimachinery/pkg/types"
+
+	eventingclient "knative.dev/eventing/pkg/client/injection/client"
```

Comment:
Should this be?:
```suggestion
	fakeeventingclient "knative.dev/eventing/pkg/client/injection/client/fake"
```

Most ctors request fakes directly instead of using that indirect injection if I'm not mistaken. e.g.

https://github.com/knative/eventing/blob/0535f9b33fe85661b6481af00355bf81e0c27bbe/pkg/reconciler/apiserversource/apiserversource_test.go#L583

https://github.com/knative/eventing/blob/0535f9b33fe85661b6481af00355bf81e0c27bbe/pkg/reconciler/inmemorychannel/controller/inmemorychannel_test.go#L363

etc.